moveit_core/ConfigExtras.cmake

```diff
@@ -1,6 +1,8 @@
 # Extras module needed for dependencies to find boost components
 
-find_package(Boost REQUIRED
+find_package(
+  Boost
+  REQUIRED
```

Comment:
Having the `REQUIRED` *anywhere* but right behind the package name seems off to me.